src/worker/ingestion/team-manager.ts

```diff
@@ -101,42 +100,35 @@ export class TeamManager {
             event: event,
             ingested: team.ingested_event,
         })
-        let shouldUpdate = this.calculateUpdates(team, event, properties)
-        if (shouldUpdate && team.__fetch_event_uuid !== eventUuid) {
```

Comment:
Based on these deletions it seems this function doesn't use `eventUuid` anymore?

Reply:
indeed! Strange that it wasn't show as unused for me and I missed it...